src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -139,6 +139,34 @@ const render = (Component: React.ComponentType) => {
   }
 };
 
+function changeFavicon(theme: 'dark' | 'light'): void {
+  const n = document.querySelector<HTMLLinkElement>('[rel="icon"][type="image/png"]');
+  if (!n) {
+    return;
+  }
+
+  const path = n.href.split('/sentry/')[0];
+  if (theme === 'dark') {
+    n.href = `${path}/sentry/images/favicon-dark.png`;
+  } else {
+    n.href = `${path}/sentry/images/favicon.png`;
+  }
+}
```

Comment:
Can we pull all this into a diff file outside of the bootstrap actually?

Reply:
I put it into a favicon.tsx file next to bootstrap, utils folder didn't feel right